backend/src/main/java/com/cs471/studentLoanSystem/login/LoginSystem.java

```diff
@@ -0,0 +1,43 @@
+package com.cs471.studentLoanSystem.login;
+
+import java.net.URI;
+
+import com.cs471.studentLoanSystem.roles.StudentInfo;
+import org.springframework.http.ResponseEntity;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+@RestController
+public class LoginSystem {
+
+    @GetMapping("/login")
+    public String loginForm(Model model) {
+        model.addAttribute("LoginInfo", new LoginInformation());
+        return "LoginInfo";
+    }
+
+    @PostMapping("/login")
+    public ResponseEntity<LoginResponse> loginForm(
+            @ModelAttribute LoginInformation information, Model model) {
+        model.addAttribute("LoginInfo", information);
+        LoginResponse response = new LoginResponse();
+        /* Fill out the response with data from the database */
+        response.setName("Andrew Gabler");
+        response.setAuthenticated(false);
```

Comment:
It looks like you are just putting in static data while you figure out the database stuff so that Ian and I have something to work with. Can you set this to true if that is the case?